src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -185,10 +185,15 @@ public function itShouldBeShippedTo(OrderInterface $order = null, $customerName,
     }
 
     /**
+     * @Then this order bill should be shipped to :customerName, :street, :postcode, :city, :countryName
```

Comment:
Is it even working? `/^(this order) bill should be shipped to "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)"$/`